src/threading.c

```diff
@@ -864,10 +864,10 @@ JL_DLLEXPORT int jl_alignment(size_t sz)
 volatile int heartbeat_enabled;
 uv_sem_t heartbeat_on_sem,              // jl_heartbeat_enable -> thread
          heartbeat_off_sem;             // thread -> jl_heartbeat_enable
-int heartbeat_interval_s,
-    n_loss_reports,
-    reset_reporting_s;
-int last_report_s, report_interval_s, n_reported;
+int heartbeat_interval,
```

Comment:
Why remove the `_s` suffix? I thought that was helpful

Reply:
I kept it in the enable function arguments. Will add them back.